cloudflare/src/framework/apiclient.rs

```diff
@@ -5,9 +5,9 @@ use crate::framework::{
 };
 use serde::Serialize;
 
-/// Synchronously sends requests to the Cloudflare API.
+/// Blocks and sends requests to the Cloudflare API.
```

Comment:
I was inspired by Reqwest which has named its APIs `reqwest::` and `reqwest::blocking::`.

I was thinking about why they did that. I like that the "default" API client is async, and to use the legacy synchronous API, you should have to type the word "blocking" because it makes you aware of the performance cost.